incubator/kafka/values.yaml

```diff
@@ -143,52 +143,100 @@ persistence:
   ##
   # storageClass:
 
-## Metrics
+jmx:
+
+  ## Rules to apply to the Kafka JMX Exporter
+  configMap:
+
+    ## Allows disabling the default configmap, note a configMap is needed
+    enabled: true
+
+    ## Allows setting values to generate confimap
+    overrideConfig: {}
+      # jmxUrl: service:jmx:rmi:///jndi/rmi://127.0.0.1:5555/jmxrmi
+      # lowercaseOutputName: true
+      # lowercaseOutputLabelNames: true
+      # ssl: false
+
+    ## Allows setting the name of the ConfigMap to use, note a configMap
+    ## with said name needs to exists for jmx stats to work
+    overrideName: ""
+
+  ## Port the jmx metrics are exposed in native jmx format, not in Prometheus format
+  port: 5555
+
+  ## JMX Whitelist Objects, can be set to control which JMX metrics are exposed.  Only whitelisted
+  ## values will be exposed via JMX Exporter.  Note: applies to default ConfigMap
+  whitelistObjectNames:
```

Comment:
If somebody wanted to expose all the metrics how would they do that? Perhaps document that in the comment.